app/pages/BeelieversAuction/MintInfo.tsx

```diff
@@ -344,8 +343,8 @@ export function MintInfo({ user, auctionInfo: { clearingPrice, auctionSize: _auc
 	const mintStarted = beelieversMint.mintStart <= +new Date();
 
 	return (
-		<Card className="lg:w-[85%] xl:w-[75%] w-full shadow-2xl border-primary/30 hover:border-primary/50 transition-all duration-300 hover:shadow-primary/10">
-			<CardContent className="p-4 lg:p-8 rounded-lg text-white flex flex-col xl:flex-row gap-6 sm:gap-8 lg:gap-12 bg-gradient-to-br from-azure-25 via-azure-20 to-azure-15">
+		<div className="card lg:w-[85%] xl:w-[75%] w-full shadow-2xl card-border border transition-all duration-300 hover:shadow-primary/10">
```

Comment:
```suggestion
		<div className="card lg:w-[85%] xl:w-[75%] w-full shadow-2xl card-border transition-all duration-300 hover:shadow-primary/10">
```

check if you need it?